.github/workflows/test-template.yml

```diff
@@ -462,13 +462,13 @@ jobs:
     timeout-minutes: 35
     runs-on: ${{ matrix.os }}
 
-    if: ${{ contains(inputs.jobsToRun, '-all-') || contains(inputs.jobsToRun, '-client-') }}
```

Comment:
should this be something like

```yaml
    if: |
      ${{
        contains(inputs.jobsToRun, '-all-') ||
        contains(inputs.jobsToRun, '-internals-') ||
        contains(inputs.jobsToRun, '-migrate-') ||
        contains(inputs.jobsToRun, '-cli-') ||
        contains(inputs.jobsToRun, '-client-')
      }}
```

?

Reply:
Thanks for the proposal, I encoded your idea in a different way